core/zap/module.go

```diff
@@ -138,6 +142,10 @@ func (m *Module) Configure(injector *dingo.Injector) {
 
 	injector.Bind(new(flamingo.Logger)).ToInstance(zapLogger)
 	flamingo.BindEventSubscriber(injector).To(shutdownEventSubscriber{})
+
+	if err := opencensus.View("flamingo/zap/errors", application.ErrorCount, view.Count()); err != nil {
```

Comment:
This should move to a global `init` function

Reply:
Done.